sdk/typescript/src/builder/Transactions.ts

```diff
@@ -61,16 +62,9 @@ export type TransactionArgument = Infer<typeof TransactionArgument>;
 
 // Transaction argument referring to an object:
 export const ObjectTransactionArgument = union([...TransactionArgumentTypes]);
```

Comment:
Do you still need theses if you don't do well known encoding? Both this and the pure version look like they're just `TransactionArgumentTypes` now

Reply:
I don't think so, I can remove them.  Wasn't sure if they were used externally

Reply:
Yeah I'd remove them, if they're used externally folks should stop that hah